src/screens/OrganizationPeople/OrganizationPeople.spec.tsx

```
@@ -36,2137 +25,1051 @@
-      window.location.search = urlObj.search;
-      window.location.hash = urlObj.hash;
-    }),
-    reload: vi.fn(),
-    pathname: '/',
-    search: '',
-    hash: '',
-    origin: 'http://localhost',
-  },
-});
+vi.mock('./AddMember', () => ({
+  default: () => <button data-testid="add-member-button">Add Member</button>,
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)